txreviewbot/_machine.py

```diff
@@ -0,0 +1,124 @@
+import attr
+from automat import MethodicalMachine
+
+
+@attr.s
+class AddLabel(object):
+    name = attr.ib()
+
+
+@attr.s
+class RemoveLabel(object):
+    name = attr.ib()
+
+
+class ReviewMachine(object):
+    _machine = MethodicalMachine()
+
+    @_machine.input()
+    def review(self, pullRequest):
```

Comment:
In the real version, no!  But for now, I'm not sure what we want out of the pull request, so I'd like to keep this as a placeholder.